cram/cram_codecs.c

```diff
@@ -750,11 +728,12 @@ cram_codec *cram_subexp_decode_init(char *data, int size,
     c->codec  = E_SUBEXP;
     c->decode = cram_subexp_decode;
     c->free   = cram_subexp_decode_free;
-    
+    c->subexp.k = -1;
+
     cp += itf8_get(cp, &c->subexp.offset);
```

Comment:
Should this be the safe itf8_get?  itf8_get always sets &c->subexp.offset therefore the c->subexp.k = -1 doesn't do anything.